### DISCUSSION TITLE
Ability to dock playtime matrix in Reaper docks

### DISCUSSION THREAD

**Discussion by stderr-enst** [helgoboss/helgobox#1055] [Ideas]
I would love to use the playtime matrix as a docked component in Reaper, since I use docks extensively.
Maybe it's not a simple feature request, since the new UI is very much its own concept and I'm not sure how well/easy it integrates with Reaper docks, but you already have the ability to hide sections of the UI, so maybe it's doable?

Probably also touches ideas of rearranging the components in the playtime ui, e.g. if the screen or Reaper dock has a more vertical/pivoted shape (9:16), instead of a more common 16:9 horizontal shape.?
But this is certainly a n-th step after adding the ability to dock the UI.

**Comment by helgoboss** [comment]
On my radar. The main challenge is the OS-specific docking. I'll need to experiment with that, it's going to be torture.

**Reply by stderr-enst** [reply]
Sure! I just thought the FR wasn't mentioned here yet and was just mentioned in one of my replies to your forum threads so far. So I try to help a bit by putting things in the same place.

Sorry to raise an OS-specific issue, portable GUIs really are a pain. :(
I didn't know Reaper handles their docks differently on other OSs, but it makes sense if I think about it for a second.
